app/admin/models/config/locations_model.php

```diff
@@ -324,6 +324,37 @@
                 'condition' => 'checked',
             ],
         ],
+        'options[limit_orders]' => [
+            'label' => 'lang:admin::lang.locations.label_limit_orders',
+            'tab' => 'lang:admin::lang.locations.text_tab_data',
+            'default' => 0,
+            'type' => 'switch'
+        ],
+        'options[limit_orders_count]' => [
+            'label' => 'lang:admin::lang.locations.label_limit_orders_count',
+            'tab' => 'lang:admin::lang.locations.text_tab_data',
+            'default' => 50,
+            'type' => 'number',
+            'span' => 'left',
+            'trigger' => [
+                'action' => 'enable',
+                'field' => 'options[limit_orders]',
+                'condition' => 'checked',
+            ],
+        ],
+        'options[limit_orders_interval]' => [
```

Comment:
Can you use the delivery/pickup time interval instead of adding a new one?

Reply:
I had thought of using those, but the reason I didn't was to allow an overall order limit slot (e.g. x orders per hour) regardless of whether they were pickup or delivery. That way you get a combined limit of pick and delivery, which seems to make more sense from kitchen perspective.